README.md

```diff
@@ -1,25 +1,18 @@
-## Build the docker image
```

Comment:
If I understand well, this build.sh script isn't useful anymore. In this case, it makes sense to remove this in the README.

However, we should not remove the other parts (migrations, adding a super user, ...) of the README except if these parts are obsolete, but in this case, we should add the new way of doing things.

Reply:
Corrected in latest commit